src/julia.h

```diff
@@ -448,7 +448,7 @@ typedef struct _jl_typemap_level_t {
     struct jl_ordereddict_t arg1;
     struct jl_ordereddict_t targ;
     jl_typemap_entry_t *linear; // union jl_typemap_t (but no more levels)
-    union jl_typemap_t any; // type at offs is Any
+    union jl_typemap_t any; // type at offs is skipped; will be split later
```

Comment:
Since you're changing the meaning of this field, can you reorder this list to show the sort order and rename it something else.

I think I would still like to keep the `any` field also, as a possible means of splitting Method tables for incremental deserialization. Scanning these tables is currently almost all of the cost there. There's fairly few instances of this type (probably a couple thousand tops), and they're already usually expected to be gigantic (several hundred to several thousand bytes).